### PR TITLE
feat: Translate and Revise "Preparation for Final Project" Module

### PR DESCRIPTION
**Pull request by stardustmeg**
#### Title of Pull Request

Translate and Revise "Preparation for Final Project" Module

#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [ ] ⚙️ Update to an existing task
- [x] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

**Comment by stardustmeg** [comment]
#1605

File: stage2/modules/final-task/README.md [hunk 1]
```diff
@@ -1,118 +1,112 @@
...
+## Final Project
 
-### Подготовка со стороны школы к выполнению курсового проекта
+### School's Preparation for Final Project Execution
 
-1. Перед выполнением курсового проекта все студенты, у которых не выполнены три последних задания подряд, отчисляются как неактивные.
+1. Before the start of the final project, all the students who have not completed the last three assignments in a row are to be expelled as inactive.
 
-2. Процесс распределения по командам полностью автоматизирован и будет проходить через https://rss-teams.web.app
+2. The team distribution process is to be done on [RS App](https://app.rs.school/course/team-distributions?course=js-fe-2023Q4).
```
Comment (valerydluski) [inline stage2/modules/final-task/README.md:7]: давай уберём ссылку на rs-app чтобы каждый курс не менять описание, напиши просто страница team-distributions в RS-App

Comment (stardustmeg) [reply stage2/modules/final-task/README.md]: Thank you! Done

**Review by valerydluski — APPROVED** [review]

**Review by haslie22 — APPROVED** [review]
LGTM 👍

**Pull request merged**